assets/js/main.js

```diff
@@ -14,26 +14,52 @@ function useToggleDarkMode() {
 }
 
 function useDarkMode() {
-  document.querySelector('html').classList.add('dark');
-  localStorage.setItem('theme', 'dark');
+  document.querySelector("html").classList.add("dark");
+  localStorage.setItem("theme", "dark");
 }
 
 function useLightMode() {
-  document.querySelector('html').classList.remove('dark');
-  localStorage.setItem('theme', 'light');
+  document.querySelector("html").classList.remove("dark");
+  localStorage.setItem("theme", "light");
 }
 
 function isDarkMode() {
-  const theme = localStorage.getItem('theme');
+  const theme = localStorage.getItem("theme");
 
-  return theme === 'dark';
+  return theme === "dark";
 }
 
-window.addEventListener('load', function() {
-  window.addEventListener('scroll', useHandleScroll);
+window.addEventListener("load", function () {
+  window.addEventListener("scroll", useHandleScroll);
 });
 
-(function() {
-  const theme = localStorage.getItem('theme');
-  theme === 'light' ? useLightMode() : useDarkMode();
+function capitalize(word) {
+  return word && word[0].toUpperCase() + word.slice(1);
+}
+
+function goToURL(url) {
+  if (!url) return;
+  window.location.href = url;
+}
+
+function decodeURIFromBreadcrumb(anchor, value) {
+  let decodeValue = decodeURI(value);
+
+  if (decodeValue.includes("-")) {
+    decodeValue = [...decodeValue.split("-")]
+      .map((word) => capitalize(word))
+      .join(" ");
+  }
+
+  anchor.innerHTML = `/ ${decodeValue}`;
+}
+
+(function () {
+  const theme = localStorage.getItem("theme");
+  theme === "light" ? useLightMode() : useDarkMode();
+
+  const breadcrumbLinks = document.querySelectorAll("a[data-breadcrumb]");
+  [...breadcrumbLinks].forEach((anchor) =>
+    decodeURIFromBreadcrumb(anchor, anchor.getAttribute("data-breadcrumb"))
+  );
```

Comment:
Until we manage to do this with Hugo's way of doing things, we will be doing this not pretty bugfix for now.

Reply:
No problem. The only thing that i would sugest is to maintain the coding style rules. Your editor should recognize eslint rules that we have there spaces and double quotes should be fixed with some non-space and single quote. Can you fix it  @pedropcruz?

Reply:
Strange that the editor itself didn't pick the coding style rules. But it seems good to me now. @miguelramos?

Reply:
Nice and clean